Dockerfile

```diff
@@ -1,57 +1,53 @@
-# syntax=docker/dockerfile:1
+# Create pipenv image to convert Pipfile to requirements.txt
+FROM python:3.9-slim as pipenv
 
-# Set base image
+# Copy Pipfile and Pipfile.lock
+COPY Pipfile Pipfile.lock ./
+
+# Install pipenv and convert to requirements.txt
+RUN pip3 install --no-cache-dir --upgrade pipenv; \
+    pipenv requirements > requirements.txt
+
+FROM python:3.9-slim as python-reqs
+
+# Copy requirements.txt from pipenv stage
+COPY --from=pipenv /requirements.txt requirements.txt
+
+# Install gcc for building python dependencies; install TCM dependencies
+RUN apt-get update; \
+    apt-get install -y gcc; \
+    pip3 install --no-cache-dir -r requirements.txt
+
+# Set base image for running TCM
 FROM python:3.9-slim
-LABEL maintainer="CollinHeist"
-LABEL description="Automated title card maker for Plex"
+LABEL maintainer="CollinHeist" \
+      description="Automated title card maker for Plex"
 
 # Set working directory, copy source into container
 WORKDIR /maker
 COPY . /maker
 
+# Copy python packages from python-reqs
+COPY --from=python-reqs /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
```

Comment:
copy in the site-packages without any of the bloat in the middle from building it.